libraries/AP_GPS/GPS_Backend.h

```diff
@@ -28,7 +29,10 @@ class AP_GPS_Backend
 
     // we declare a virtual destructor so that GPS drivers can
     // override with a custom destructor if need be.
-    virtual ~AP_GPS_Backend(void) {}
+    virtual ~AP_GPS_Backend(void) {
```

Comment:
are all GPS backends calling the parent destructor correctly in their own destructors?